### PR TITLE
Add request for getting combined result from a chain of paginated responses

### PR DESCRIPTION
**Pull request by onevcat**

### CODE REVIEW

File: LineSDK/LineSDK/Networking/Client/ChainedPaginatedRequest.swift [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+        pipelines.append(.redirector(BadHTTPStatusRedirector(valid: 200..<300)))
+        pipelines.append(.redirector(PaginatedParseRedirector(request: self, parser: defaultJSONParser)))
+        pipelines.append(.terminator(PaginatedResultTerminator(request: self)))
+
+        return pipelines
+    }
+}
+
+// Parse a intermediate response (single response) of a `ChainedPaginatedRequest`.
```
Comment (onevcat) [inline LineSDK/LineSDK/Networking/Client/ChainedPaginatedRequest.swift]:
```suggestion
// Parse an intermediate response (single response) of a `ChainedPaginatedRequest`.
```

File: LineSDK/LineSDK/Graph/Request/GetFriendsRequest.swift [hunk 2]
```diff
@@ -68,3 +68,11 @@ public struct GetFriendsRequest: Request {
         public let pageToken: String?
     }
 }
+
+extension GetFriendsRequest: Sortable {
+    var sortValue: String? { return sort?.rawValue }
```
Comment (eJamesLin) [inline LineSDK/LineSDK/Graph/Request/GetFriendsRequest.swift]: seems `sortValue` here means a `sortParameter` ?

Comment (onevcat) [reply LineSDK/LineSDK/Graph/Request/GetFriendsRequest.swift]: Yes! That name is better.

**Comment by onevcat** [comment]
@eJamesLin Renamed in the latest commit.

**Pull request merged**